comex-main/src/main/java/br/com/alura/comex/processadores/ProcessadorDeCsv.java

```diff
@@ -0,0 +1,55 @@
+package br.com.alura.comex.processadores;
+
+import java.io.IOException;
+import java.math.BigDecimal;
+import java.net.URISyntaxException;
+import java.net.URL;
+import java.nio.file.Path;
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Scanner;
+
+import br.com.alura.comex.*;
+
+public class ProcessadorDeCsv {
+
+	public List<Pedido> listarPedidos(String pedidosCSV) {
+
+		ArrayList<Pedido> pedidos = new ArrayList<>();
+
+		try {
+			URL recursoCSV = ClassLoader.getSystemResource(pedidosCSV);
+			Path caminhoDoArquivo = caminhoDoArquivo = Path.of(recursoCSV.toURI());
```

Comment:
Remover atribuição desnecessária que eu havia deixado no código:

```suggestion
			Path caminhoDoArquivo = Path.of(recursoCSV.toURI());
```